stat.go

```diff
@@ -648,6 +648,71 @@ func KullbackLeibler(p, q []float64) float64 {
 	return kl
 }
 
+// OrdinaryLinearRegression computes the simple linear regression of x and y.
+// It returns the values for \hat{\alpha}, \hat{\beta} and the coefficient
+// of determination, r. If origin is true, the regression is forced to pass
+// through the origin.
+// The lengths of x and y must be equal. If weights is nil then all of the
```

Comment:
Ahh, I see. You are meaning for paras. Note that that is not applied uniformly here. Maybe that should be fixed.